blog/content/introtogonum.md

```diff
@@ -0,0 +1,145 @@
+# The Gonum Numeric Computing Packages
+
+## Gonum -- The Basics
+
+Gonum is a set of packages designed to make writing numeric and scientific
+algorithms productive, performant, and scalable.
+
+Gonum is a set of packages written in the 
+[Go programming language](https://www.golang.org) (more on Go later), and is
+not a language on its own.
+This makes Gonum more similar to [numpy](https://www.numpy.org) and
+[scipy](https://www.scipy.org), libraries built on top of
+[python](https://www.python.org), and different from
+[Julia](https://julialang.org/) and [Matlab](https://www.mathworks.com/)
+which are full programming languages.
+
+Gonum contains libraries for [matrices and linear algebra](http://godoc.org/gonum.org/v1/gonum/mat);
```

Comment:
you should use `https` instead of plain `http`. (similar for all other godoc.org links)